diff -up PyQt5_gpl-5.11.2/configure.py.sip_check PyQt5_gpl-5.11.2/configure.py
--- PyQt5_gpl-5.11.2/configure.py.sip_check	2018-07-11 14:10:21.751437714 -0500
+++ PyQt5_gpl-5.11.2/configure.py	2018-07-11 14:12:13.035482279 -0500
@@ -647,7 +647,7 @@ class TargetConfiguration:
         try:
             from PyQt5 import sip
         except ImportError:
-            error(
+            inform(
                     "Unable to import PyQt5.sip.  Make sure you have "
                     "configured SIP to create a private copy of the sip "
                     "module.")
@@ -2919,7 +2919,7 @@ def check_sip(target_config):
     try:
         from PyQt5 import sip
     except ImportError:
-        error(
+        inform(
                 "Unable to import PyQt5.sip.  Make sure you have configured "
                 "SIP to create a private copy of the sip module.")
 
